src/Filesystem/Filesystem.php

```diff
@@ -15,10 +15,82 @@
 
 use Symfony\Component\Filesystem\Filesystem as BaseFilesystem;
 
-final class Filesystem extends BaseFilesystem implements FilesystemInterface
+final class Filesystem implements FilesystemInterface
```

Comment:
We've duplicated the contract of this class to run `exists` twice and `getFileContents` once... We should deprecate this class and propose better abstraction (suited for our usecase)